src/components/card-view/card-view.scss

```diff
@@ -0,0 +1,104 @@
+.cardView {
+  font-size: 11px;
+  font-family: 'Montserrat', sans-serif;
+
+  .case {
+    margin: 5px 7px;
+    padding: 2px;
+    border-radius: 5px;
+    box-shadow: inset 0 2px 3px rgba(0,0,0,0.3), 0 2px 0 rgba(255,255,255,0.8);
+
+    &:first-child {
+      box-shadow: none;
+    }
+
+    .title {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      text-align: left;
+      margin: 0 0 10px;
+      padding: 3px;
+      border-radius: 4px;
+      border: solid 1px #cee9fb;
+      background-color: #fff;
+      font-weight: 600;
+      color: var(--text);
```

Comment:
Should we add this variable for the text color?

Reply:
No, I just removed the references: https://github.com/concord-consortium/multidata-plugin/pull/25/commits/9e19b24ab50ce45d944dfcc6d99ae38481d8dc21